lib/puppet/provider/chocolateyfeature/windows.rb

```diff
@@ -0,0 +1,138 @@
+require 'puppet/type'
+require 'pathname'
+require 'rexml/document'
+
+Puppet::Type.type(:chocolateyfeature).provide(:windows) do
+  confine :operatingsystem => :windows
+  defaultfor :operatingsystem => :windows
+
+  require Pathname.new(__FILE__).dirname + '../../../' + 'puppet_x/chocolatey/chocolatey_common'
+  include PuppetX::Chocolatey::ChocolateyCommon
+
+  MINIMUM_SUPPORTED_CHOCO_VERSION = '0.9.9.0'
```

Comment:
I think this constant definition might clash with the one of the same name in the sources code.